### ISSUE TITLE
Fix auto-generation of outstanding issues to respect XDG configuration

### ISSUE THREAD

**Issue by bkuhlmann** [rubocop/rubocop#12729]
## Expected behavior

Hello. :wave: First, thanks for adding *local* XDG support in 1.61.0. This is exciting to see.

Unfortunately, I ran into an issue with regenerating my TODO list after picking up the changes in 1.61.0 where running `rubocop --auto-gen-config` will not respect use of project specific XDG configuration and rebuild the TODO list in the root of the project instead.

## Actual behavior

Here's my configuration structure:

``` 
.config
└── rubocop
   ├── config.yml
   └── issues.yml
```

If it helps, I have this line in `config.yml`:

``` 
inherit_from:
  - issues.yml
```

...which works great because when I have RuboCop analyze my project, the XDG configuration is picked up and so is the issues list (er, TODO list).

The following is the debug information from running `rubocop --debug --auto-gen-config`

<details>
  <summary>Debug Information</summary>

⚠️ I had to truncate this output due to GitHub issue limit of 65536 due to all the new cops I have enabled.

``` bash
Added inheritance from `.rubocop_todo.yml` in `.rubocop.yml`.
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
Default configuration from /Users/bkuhlmann/.cache/frum/versions/3.3.0/lib/ruby/gems/3.3.0/gems/rubocop-1.61.0/config/default.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

Gemspec/DeprecatedAttributeAssignment: # new in 1.30
  Enabled: true
Gemspec/DevelopmentDependencies: # new in 1.44
  Enabled: true
Gemspec/RequireMFA: # new in 1.23
  Enabled: true
Layout/LineContinuationLeadingSpace: # new in 1.31
  Enabled: true
Layout/LineContinuationSpacing: # new in 1.31
  Enabled: true
Layout/LineEndStringConcatenationIndentation: # new in 1.18
  Enabled: true
Layout/SpaceBeforeBrackets: # new in 1.7
  Enabled: true
Lint/AmbiguousAssignment: # new in 1.7
  Enabled: true
Lint/AmbiguousOperatorPrecedence: # new in 1.21
  Enabled: true
Lint/AmbiguousRange: # new in 1.19
  Enabled: true
Lint/ConstantOverwrittenInRescue: # new in 1.31
  Enabled: true
Lint/DeprecatedConstants: # new in 1.8
  Enabled: true
Lint/DuplicateBranch: # new in 1.3
  Enabled: true
Lint/DuplicateMagicComment: # new in 1.37
  Enabled: true
Lint/DuplicateMatchPattern: # new in 1.50
  Enabled: true
Lint/DuplicateRegexpCharacterClassElement: # new in 1.1
  Enabled: true
Lint/EmptyBlock: # new in 1.1
  Enabled: true
Lint/EmptyClass: # new in 1.3
  Enabled: true
Lint/EmptyInPattern: # new in 1.16
  Enabled: true
Lint/IncompatibleIoSelectWithFiberScheduler: # new in 1.21
  Enabled: true
Lint/ItWithoutArgumentsInBlock: # new in 1.59
  Enabled: true
Lint/LambdaWithoutLiteralBlock: # new in 1.8
  Enabled: true
Lint/LiteralAssignmentInCondition: # new in 1.58
  Enabled: true
Lint/MixedCaseRange: # new in 1.53
  Enabled: true
Lint/NoReturnInBeginEndBlocks: # new in 1.2
  Enabled: true
Lint/NonAtomicFileOperation: # new in 1.31
  Enabled: true
Lint/NumberedParameterAssignment: # new in 1.9
  Enabled: true
Lint/OrAssignmentToConstant: # new in 1.9
  Enabled: true
Lint/RedundantDirGlobSort: # new in 1.8
  Enabled: true
Lint/RedundantRegexpQuantifiers: # new in 1.53
  Enabled: true
Lint/RefinementImportMethods: # new in 1.27
  Enabled: true
Lint/RequireRangeParentheses: # new in 1.32
  Enabled: true
Lint/RequireRelativeSelfPath: # new in 1.22
  Enabled: true
Lint/SymbolConversion: # new in 1.9
  Enabled: true
Lint/ToEnumArguments: # new in 1.1
  Enabled: true
Lint/TripleQuotes: # new in 1.9
  Enabled: true
Lint/UnexpectedBlockArity: # new in 1.5
  Enabled: true
Lint/UnmodifiedReduceAccumulator: # new in 1.1
  Enabled: true
Lint/UselessRescue: # new in 1.43
  Enabled: true
Lint/UselessRuby2Keywords: # new in 1.23
  Enabled: true
Metrics/CollectionLiteralLength: # new in 1.47
  Enabled: true
Naming/BlockForwarding: # new in 1.24
  Enabled: true
Security/CompoundHash: # new in 1.28
  Enabled: true
Security/IoMethods: # new in 1.22
  Enabled: true
Style/ArgumentsForwarding: # new in 1.1
  Enabled: true
Style/ArrayIntersect: # new in 1.40
  Enabled: true
Style/CollectionCompact: # new in 1.2
  Enabled: true
Style/ComparableClamp: # new in 1.44
  Enabled: true
Style/ConcatArrayLiterals: # new in 1.41
  Enabled: true
Style/DataInheritance: # new in 1.49
  Enabled: true
Style/DirEmpty: # new in 1.48
  Enabled: true
Style/DocumentDynamicEvalDefinition: # new in 1.1
  Enabled: true
Style/EmptyHeredoc: # new in 1.32
  Enabled: true
Style/EndlessMethod: # new in 1.8
  Enabled: true
Style/EnvHome: # new in 1.29
  Enabled: true
Style/ExactRegexpMatch: # new in 1.51
  Enabled: true
Style/FetchEnvVar: # new in 1.28
  Enabled: true
Style/FileEmpty: # new in 1.48
  Enabled: true
Style/FileRead: # new in 1.24
  Enabled: true
Style/FileWrite: # new in 1.24
  Enabled: true
Style/HashConversion: # new in 1.10
  Enabled: true
Style/HashExcept: # new in 1.7
  Enabled: true
Style/IfWithBooleanLiteralBranches: # new in 1.9
  Enabled: true
Style/InPatternThen: # new in 1.16
  Enabled: true
Style/MagicCommentFormat: # new in 1.35
  Enabled: true
Style/MapCompactWithConditionalBlock: # new in 1.30
  Enabled: true
Style/MapToHash: # new in 1.24
  Enabled: true
Style/MapToSet: # new in 1.42
  Enabled: true
Style/MinMaxComparison: # new in 1.42
  Enabled: true
Style/MultilineInPatternThen: # new in 1.16
  Enabled: true
Style/NegatedIfElseCondition: # new in 1.2
  Enabled: true
Style/NestedFileDirname: # new in 1.26
  Enabled: true
Style/NilLambda: # new in 1.3
  Enabled: true
Style/NumberedParameters: # new in 1.22
  Enabled: true
Style/NumberedParametersLimit: # new in 1.22
  Enabled: true
Style/ObjectThen: # new in 1.28
  Enabled: true
Style/OpenStructUse: # new in 1.23
  Enabled: true
Style/OperatorMethodCall: # new in 1.37
  Enabled: true
Style/QuotedSymbols: # new in 1.16
  Enabled: true
Style/RedundantArgument: # new in 1.4
  Enabled: true
Style/RedundantArrayConstructor: # new in 1.52
  Enabled: true
Style/RedundantConstantBase: # new in 1.40
  Enabled: true
Style/RedundantCurrentDirectoryInPath: # new in 1.53
  Enabled: true
Style/RedundantDoubleSplatHashBraces: # new in 1.41
  Enabled: true
Style/RedundantEach: # new in 1.38
  Enabled: true
Style/RedundantFilterChain: # new in 1.52
  Enabled: true
Style/RedundantHeredocDelimiterQuotes: # new in 1.45
  Enabled: true
Style/RedundantInitialize: # new in 1.27
  Enabled: true
Style/RedundantLineContinuation: # new in 1.49
  Enabled: true
Style/RedundantRegexpArgument: # new in 1.53
  Enabled: true
Style/RedundantRegexpConstructor: # new in 1.52
  Enabled: true
Style/RedundantSelfAssignmentBranch: # new in 1.19
  Enabled: true
Style/RedundantStringEscape: # new in 1.37
  Enabled: true
Style/ReturnNilInPredicateMethodDefinition: # new in 1.53
  Enabled: true
Style/SelectByRegexp: # new in 1.22
  Enabled: true
Style/SingleLineDoEndBlock: # new in 1.57
  Enabled: true
Style/StringChars: # new in 1.12
  Enabled: true
Style/SuperWithArgsParentheses: # new in 1.58
  Enabled: true
Style/SwapValues: # new in 1.1
  Enabled: true
Style/YAMLFileRead: # new in 1.53
  Enabled: true
For more information: https://docs.rubocop.org/rubocop/versioning.html
Phase 1 of 2: run Layout/LineLength cop
Inspecting 46 files
Scanning /Users/bkuhlmann/Engineering/OSS/navigator/Gemfile
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/Guardfile
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/Rakefile
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/app/helpers/navigator/navigation_helper.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/console
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/guard
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/rake
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/rspec
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/rubocop
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator/engine.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator/menu.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator/tag.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator/tag_activator.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/navigator.gemspec
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/app/helpers/navigator/navigation_helper_spec.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/features/navigator_spec.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/lib/navigator/menu_spec.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/lib/navigator/tag_activator_spec.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/lib/navigator/tag_spec.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/rails_helper.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/spec_helper.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/Rakefile
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/controllers/application_controller.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/controllers/home_controller.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/controllers/pages_controller.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/controllers/posts_controller.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/helpers/application_helper.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/helpers/layout_helper.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/jobs/application_job.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/mailers/application_mailer.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/models/application_record.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/bin/rails
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/bin/rake
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config.ru
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/application.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/boot.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/environment.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/environments/development.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/environments/production.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/environments/test.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/initializers/filter_parameter_logging.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/puma.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/routes.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/shared_contexts/temp_dir.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/shared_examples/tag_prefix.rb
.

46 files inspected, no offenses detected
Created .rubocop_todo.yml.
Phase 2 of 2: run all cops
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

# (truncated for brevity)
For more information: https://docs.rubocop.org/rubocop/versioning.html
Inspecting 46 files
Scanning /Users/bkuhlmann/Engineering/OSS/navigator/Gemfile
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/Guardfile
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/Rakefile
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/app/helpers/navigator/navigation_helper.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/console
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/guard
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/rake
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/rspec
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/bin/rubocop
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator/engine.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator/menu.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator/tag.rb
WScanning /Users/bkuhlmann/Engineering/OSS/navigator/lib/navigator/tag_activator.rb
WScanning /Users/bkuhlmann/Engineering/OSS/navigator/navigator.gemspec
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/app/helpers/navigator/navigation_helper_spec.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/features/navigator_spec.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/lib/navigator/menu_spec.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/lib/navigator/tag_activator_spec.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/lib/navigator/tag_spec.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/rails_helper.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/spec_helper.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/Rakefile
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/controllers/application_controller.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/controllers/home_controller.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/controllers/pages_controller.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/controllers/posts_controller.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/helpers/application_helper.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/helpers/layout_helper.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/jobs/application_job.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/mailers/application_mailer.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/app/models/application_record.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/bin/rails
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/bin/rake
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config.ru
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/application.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/boot.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/environment.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/environments/development.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/environments/production.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/environments/test.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/initializers/filter_parameter_logging.rb
.Scanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/puma.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/dummy/config/routes.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/shared_contexts/temp_dir.rb
CScanning /Users/bkuhlmann/Engineering/OSS/navigator/spec/support/shared_examples/tag_prefix.rb
C

46 files inspected, 763 offenses detected, 729 offenses autocorrectable
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

# (truncated for brevity)
For more information: https://docs.rubocop.org/rubocop/versioning.html
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

# (truncated for brevity)
For more information: https://docs.rubocop.org/rubocop/versioning.html
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

# (truncated for brevity)
For more information: https://docs.rubocop.org/rubocop/versioning.html
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

# (truncated for brevity)
For more information: https://docs.rubocop.org/rubocop/versioning.html
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

# (truncated for brevity)
For more information: https://docs.rubocop.org/rubocop/versioning.html
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

# (truncated for brevity)
For more information: https://docs.rubocop.org/rubocop/versioning.html
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

# (truncated for brevity)
For more information: https://docs.rubocop.org/rubocop/versioning.html
For /Users/bkuhlmann/Engineering/OSS/navigator: configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop.yml
Inheriting configuration from /Users/bkuhlmann/Engineering/OSS/navigator/.rubocop_todo.yml
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

# (truncated for brevity)
For more information: https://docs.rubocop.org/rubocop/versioning.html
Created .rubocop_todo.yml.
Finished in 1.2675880000460893 seconds
```
</details>

## Steps to reproduce the problem

The quickest way to reproduce the problem is to:

- Create a Ruby style violation.
- Run `rubocop --auto-gen-config` to generate a TODO list.
- Move the `.rubocop_todo.yml` into `.config/rubocop/issues.yml`
- Update your `.config/rubocop/config.yml` to at least have this line in it:

``` yaml
inherit_from:
  - issues.yml
```

Now when you run `rubocop --auto-gen-config`, notice that you have two files show up in the root of your project:

``` 
.rubocop.yml
.rubocop_todo.yml
```

Normally, this would be expected but with the new XDG support, we have to manally move/rename these files back into the `.config/rubocop` folder in order to update our XDG configuration.

I think it'd be great if RuboCop to respect the local XDG configuration and do all this automatically.

## RuboCop version

```
1.61.0 (using Parser 3.3.0.5, rubocop-ast 1.30.0, running on ruby 3.3.0) [arm64-darwin23.3.0]
```

**Comment by bbatsov** [comment]
Closing as stale — no comments or reactions in 2 years. If this is still an issue for someone, please reopen with a reproducible example.
